torch/_utils_internal.py

```diff
@@ -54,3 +54,4 @@ def get_source_lines_and_file(obj):
 
 TEST_MASTER_ADDR = '127.0.0.1'
 TEST_MASTER_PORT = 29500
+USE_RTLD_GLOBAL_WITH_LIBTORCH = False
```

Comment:
Why do we have this constant if it's always false? Is this so that you can patch it in fbcode?

Reply:
Yep, fbcode shenanigans